native/packages/react-native-bpk-component-button/src/BpkButton.js

```diff
@@ -18,152 +18,45 @@
 
 import {
   View,
-  TouchableHighlight,
-  TouchableNativeFeedback,
   Platform,
 } from 'react-native';
 import React from 'react';
 import PropTypes from 'prop-types';
-import LinearGradient from 'react-native-linear-gradient';
-import difference from 'lodash/difference';
 
 import { withTheme } from 'react-native-bpk-theming';
 import BpkIcon from 'react-native-bpk-component-icon';
 import BpkText from 'react-native-bpk-component-text';
 
-import styles, { colorWhite } from './BpkButton-styles';
-
+import {
+  isTypeThemeable,
+  themeAttributesSupplied,
+  getStyleForElement,
+  iconPropType,
+  themePropType,
+  textStyle,
+} from './utils';
+
+import BpkButtonAndroidContainer from './layout/BpkButtonContainer.android';
+import BpkButtonIosContainer from './layout/BpkButtonContainer.ios';
+
+const BpkButtonContainer = Platform.OS === 'android' ? BpkButtonAndroidContainer : BpkButtonIosContainer;
```

Comment:
Shouldn't we be relying on the RN packager to ensure that the right code is loaded?

Reply:
yes indeed, I've changed it but I revert it back somehow 😅